### PR TITLE
Add a promo for the Autumn statement 2016

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/gZJa7bWK

## Description

Replace the GOV.UK blogs promo with a promo for the Autumn statement on the GOV.UK homepage.

### Before
screen shot 2016-11-22 at 11 14 21

### After
screen shot 2016-11-22 at 11 13 48

### CODE REVIEW

File: app/views/homepage/index.html.erb [hunk 1]
```diff
@@ -172,9 +172,9 @@
             </div>
             <div class="promo-image">
               <div class="promo-content">
-                <a href="https://www.blog.gov.uk"><%= image_tag 'homepage/govuk-blogs.jpg', alt: 'GOV.UK blogs' %></a>
-                <h3>GOV.UK blogs</h3>
-                <p><a href="https://www.blog.gov.uk">Search the list</a> of GOV.UK blogs, and find one to match your interest.</p>
+                <a href="/government/topical-events/autumn-statement-2016"><%= image_tag 'homepage/number-11.jpg', alt: 'Autumn Statement 2016' %></a>
+                <h3>Autumn Statement 2016</h3>
+                <p>Read all the news about the <a href="/government/topical-events/autumn-statement-2016">Chancellor's statement on 22 November</a>.</p>
```
Comment (fofr) [inline app/views/homepage/index.html.erb]: Can we use a smart quote for `Chancellor’s`?

Comment (leenagupte) [inline app/views/homepage/index.html.erb]: @fofr I've changed the quote and fixed-up. Thanks!

**Comment by dsingleton** [comment]
Can we remove the `govuk-blogs.jpg` image too, if it's not used anymore?

**Comment by selfthinker** [comment]
@dsingleton, this promo will only be live for a week. After that the old blog promo will be used again.

**Pull request merged**